dockstore-webservice/src/main/java/io/dockstore/webservice/core/Tool.java

```diff
@@ -61,7 +61,7 @@
         "This describes one entry in the dockstore. Logically, this currently means one tuple of registry (either quay or docker hub), organization, image name, and toolname which can be\n"
                 + " * associated with CWL and Dockerfile documents")
 @Entity
-@Table(uniqueConstraints = @UniqueConstraint(columnNames = { "registry", "namespace", "name", "toolname" }))
+@Table(uniqueConstraints = @UniqueConstraint(name = "ukbq5vy17y4ocaist3d3r3imcus", columnNames = { "registry", "namespace", "name", "toolname" }))
```

Comment:
constraint name

Reply:
I don't understand it, but I encountered issues dropping the constraint and re-adding it.
I think I'll limit this PR to matching existing structure and add improvements in a subsequent PR